common/remote/sockfile.cpp

```diff
@@ -3984,8 +3999,9 @@ class CRemoteDiskBaseActivity : public CSimpleInterfaceOf<IRemoteActivity>, impl
 public:
     IMPLEMENT_IINTERFACE_USING(CSimpleInterfaceOf<IRemoteActivity>)
 
-    CRemoteDiskBaseActivity()
+    CRemoteDiskBaseActivity(IPropertyTree &config)
```

Comment:
why not IPropertyTree * config

Reply:
+ consistency with existing derivatives
+ when it is illegal for object/interface to be null, reference is clearer.
+ preference when purely used and not owned (not link counted etc).